app/styles/ui/_popover.scss

```diff
@@ -40,6 +40,31 @@
   }
 }
 
+.popover-component.popover-caret-top-left {
+  &::before,
+  &::after {
+    position: absolute;
+    left: 20px;
+    display: inline-block;
+    content: '';
+    pointer-events: none;
+  }
+
+  &::before {
+    top: -16px;
+    margin-right: -9px;
+    border: 8px solid transparent;
+    border-bottom-color: var(--box-border-color);
+  }
+
+  &::after {
+    top: -14px;
+    margin-right: -8px;
+    border: 7px solid transparent;
+    border-bottom-color: var(--background-color);
+  }
```

Comment:
Yeah. I see what you are getting at; But, I think it would require potentially creating popover-caret class as distinct form the placement class and I am not inclined to tear that apart with this pr.